api/python/cellxgene_census/release_process.md

```diff
@@ -15,7 +15,7 @@ While not strictly required, this process assumes you have met the following pre
 - You have write access to the `chanzuckerberg/cellxgene-census` repo
 - You have an account on pypi.org and test.pypi.org, both with access to the cellxgene_census project
 - You have the Github CLI tool (`gh`) installed. See [documentation](https://cli.github.com/).
-- You have the `pipx` CLI tool installed. See [documentation](https://pypa.github.io/pipx/).
+- You have the `pipx` CLI tool installed. See [documentation](https://pypa.github.io/pipx/). Note that you will need create an API token on both test.pypi.org and pypi.org accounts and use this to authenticate with `pipx`. See https://pypi.org/help/#apitoken for more information.
```

Comment:
the API token has nothing to do with `pipx`.  The token is required by `twine` (or really by PyPi)

Reply:
done